docs/admin/admin-ui/scripts.md

```
@@ -25,8 +25,8 @@ The Janssen Authentication Server leverages custom scripts, which, when implemen
 
 ## Script properties
 
-Each script presents a few default properties where values can be set 
+Each script presents a few default properties where values can be set
 according to the requirement. Details can be found [here](https://docs.jans.io/stable/janssen-server/developer/scripts/#configurable-properties-of-a-custom-script).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)